db/migrate/20220919183401_drop_user_role.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class DropUserRole < ActiveRecord::Migration[7.0]
+  def change
+    execute 'DROP TABLE user_roles CASCADE'
```

Comment:
why are you using raw sql in migration

Reply:
can't you use drop_table :user_roles

Reply:
UserRoles had associations with user and i couldn't migrate while using drop_table because it was throwing an error